packages/types/src/payment/mutations.ts

```diff
@@ -35,21 +34,24 @@ export interface UpdatePaymentCollectionDTO
    * The ID of the payment collection.
    */
   id: string
+}
 
-  /**
-   * The authorized amount of the payment collection.
-   */
-  authorized_amount?: number
-
-  /**
-   * The refunded amount of the payment collection.
-   */
-  refunded_amount?: number
+export interface UpsertPaymentCollectionDTO {
+  id?: string
+  region_id?: string
+  currency_code?: string
+  amount?: number
+  metadata?: Record<string, unknown>
+}
 
-  /**
-   * The status of the payment collection.
-   */
-  status?: PaymentCollectionStatus
+/**
+ * The attributes to update in the payment collection.
+ */
+export interface PaymentCollectionUpdatableFields {
```

Comment:
I think we can call these `UpdatePaymentCollectionDTO` to be consistent with create and upsert, and really the mean the same thing.